src/party_menu.c

```diff
@@ -2555,8 +2559,8 @@ static void UpdateCurrentPartySelection(s8 *ptr, s8 movementDir)
     if (*ptr != slot)
     {
         PlaySE(SE_SELECT);
-        sub_81B0FCC(slot, 0);
-        sub_81B0FCC(*ptr, 1);
+        UpdatePartyMenuSlotSelected(slot, FALSE); // deselect old slot
+        UpdatePartyMenuSlotSelected(*ptr, TRUE); //select new slot
```

Comment:
In code, please put comments on their own line. So below `PlayeSE` you would put:
```C
// Deselect old slot, then select new slot.
UpdatePartyMenuSlotSelected(...
```

Reply:
What's the benefit of doing that over the very clear comments added here? There's an immediate loss in terms of locality by moving comments away from their relevant locations, even if it is by a couple of lines...

Reply:
It's not moving the comments away; these two comments just become one comment explaining the overall action. Comments are supposed to explain what code is trying to do, not explain what a particular line does. Individual lines should speak for themselves, unless they're very long.

Reply:
That seems like style for the sake of style with zero improvement on anything (readability, cleanness, understandability of the code) to me.